app/models/nufs_account.rb

```diff
@@ -6,8 +6,13 @@ class NufsAccount < Account
 
   after_find :load_components
 
-  def set_expires_at!
+  def set_expires_at
     self.expires_at = ValidatorFactory.instance(account_number).latest_expiration
+    # If this Validator instance has an error, suppress it. The errors
+    # will be raised later.
+    rescue ValidatorError, AccountNumberFormatError
+      # Prevent the expires_at validation error message from appearing on the front end
+      self.expires_at ||= Time.current
```

Comment:
Yes, they're pretty much the same. https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/time/calculations.rb#L36

I'm partial to `Time.current`, but we have lots of older references to `Time.zone.now`.